src/registrar/assets/src/js/getgov/main.js

```diff
@@ -44,5 +36,7 @@ initMembersTable();
 initMemberDomainsTable();
 initEditMemberDomainsTable();
 
-initPortfolioMemberPageToggle();
+// Init the portfolio new member page
+initPortfolioMemberPageRadio();
```

Comment:
I feel like we should start bundling inits()

Reply:
Yes, I agree. On a few of the other tickets we have worked on recently, we have consolidated inits to one init per page in the main.js.